### PR TITLE
docs: update file path in next-auth.md

### PR DESCRIPTION
**Pull request by TawalMc**
I change some file path (for code example) which are not correct like:
- server/api/trpc.ts and types/next-auth.d.ts instead of server/auth.ts

Closes #<issue>

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] I performed a functional test on my final commit

---

## Changelog

_[Short description of what has changed]_

---

## Screenshots

_[Screenshots]_

💯

### CODE REVIEW

File: www/src/pages/en/usage/next-auth.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ const User = () => {
 
 Create T3 App is configured to utilise the [session callback](https://next-auth.js.org/configuration/callbacks#session-callback) in the NextAuth.js config to include the user's ID within the `session` object.
 
-```ts:server/auth.ts
+```ts:/pages/api/auth/[...nextauth].ts
```
Comment (juliusmarminge) [inline www/src/pages/en/usage/next-auth.md:64]: `server/auth.ts` is the correct path

File: www/src/pages/en/usage/next-auth.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ callbacks: {
 
 This is coupled with a type declaration file to make sure the `user.id` is typed when accessed on the `session` object. Read more about [`Module Augmentation`](https://next-auth.js.org/getting-started/typescript#module-augmentation) on NextAuth.js's docs.
 
-```ts:server/auth.ts
+```ts:types/next-auth.d.ts
```
Comment (juliusmarminge) [inline www/src/pages/en/usage/next-auth.md:77]: again, `server/auth.ts` is correct

File: www/src/pages/en/usage/next-auth.md [hunk 3]
```diff
@@ -96,7 +96,7 @@ This is done in a two step process:
 
 1. Grab the session from the request headers using the [`getServerSession`](https://next-auth.js.org/configuration/nextjs#getServerSession) function. The advantage of using `getServerSession` instead of the regular `getSession` is that it's a server-side only function and doesn't trigger unnecessary fetch calls. `create-t3-app` creates a helper function that abstracts this peculiar API away so that you don't need to import both your NextAuth.js options as well as the `getServerSession` function every time you need to access the session.
 
-```ts:server/auth.ts
+```ts:server/api/trpc.ts
```
Comment (juliusmarminge) [inline www/src/pages/en/usage/next-auth.md:99]: this is also already correct with `server/auth.ts`

**Review by juliusmarminge — COMMENTED** [review]
Seems like you are on an old version cause all these paths are already correct. See here: https://github.com/t3-oss/create-t3-app/blob/next/cli/template/extras/src/server/auth/with-prisma.ts

**Comment by TawalMc** [comment]
Ok ok. Sorry, I'm using the v7.1...

**Pull request closed without merging**